css-two-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+body {
+  text-align: center;
+  margin: 0 auto;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: large;
+}
+
+.container {
+  letter-spacing: 1px;
+  line-height: 30px;
+  justify-content: space-evenly;
```

Comment:
`justify-content` only works when used alongside `display: flex`. Since `display: flex` isn't being applied, this should be removed.